app/uk/gov/hmrc/taxhistory/auditable/Auditable.scala

```diff
@@ -17,21 +17,20 @@
 package uk.gov.hmrc.taxhistory.auditable
 
 import uk.gov.hmrc.http.HeaderCarrier
-import uk.gov.hmrc.play.audit.AuditExtensions
-import uk.gov.hmrc.play.audit.model.{Audit, DataEvent}
-import uk.gov.hmrc.play.config.AppName
+import uk.gov.hmrc.taxhistory.model.audit.{DataEventAuditType, DataEventDetail, DataEventTransaction}
+import scala.concurrent.Future
 
-trait Auditable extends AppName{
+trait Auditable {
```

Comment:
Do you think there is a need to break the Auditing functionality into a trait and Impl? Rather than a singleton class? What do you think?